File: serve/src/main/java/com/example/serve/entity/employee/Employee.java
package com.example.serve.entity.employee;

import com.baomidou.mybatisplus.annotation.*;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

import java.io.Serializable;
import java.util.Date;
@Data
@TableName("employee")
public class Employee implements Serializable {

    @TableId(type = IdType.ASSIGN_ID)
    private Long id;

    private String name;

    private String phone;

    private String address;

    private Date date;

    @TableField(fill = FieldFill.INSERT)
    private Date createTime;

    @TableField(fill = FieldFill.INSERT_UPDATE)
    private Date updateTime;

}

File: admin/src/types/employee/employee.ts
/**
 * 员工信息
 */
export interface Employee {
  /**
   * 地址
   */
  address?: string
  /**
   * 创建时间
   */
  createTime?: Date
  /**
   * 入职日期
   */
  date?: Date
  /**
   * 员工id
   */
  id?: string
  /**
   * 员工姓名
   */
  name?: string
  /**
   * 电话
   */
  phone?: string
  /**
   * 更新时间
   */
  updateTime?: Date
}

/**
 * 分页参数
 */
export interface EmployeeSearch extends PageParams {
  /**
   * 地址
   */
  address?: string
  /**
   * 员工姓名
   */
  name?: string
  /**
   * 电话
   */
  phone?: string
}

/**
 * 表单参数
 */
export interface EmployeeForm {
  /**
   * 地址
   */
  address?: string
  /**
   * 入职日期
   */
  date?: Date
  /**
   * 员工id
   */
  id?: string
  /**
   * 员工姓名
   */
  name?: string
  /**
   * 电话
   */
  phone?: string
}
